packages/containers/src/SubHeaderBar/SubHeaderBar.selectors.js

```diff
@@ -8,12 +8,3 @@ import { DEFAULT_STATE, DISPLAY_NAME } from './SubHeaderBar.container';
 export function getComponentState(state, idComponent) {
 	return state.cmf.components.getIn([DISPLAY_NAME, idComponent], DEFAULT_STATE);
 }
-
-/**
- * Return the editMode attr from the state of subheader bar id given.
- * @param {object} state
- * @param {string} idComponent
- */
-export function getEditMode(state, idComponent) {
```

Comment:
Be careful this is a breaking change. This can impact the apps. Please communicate on this after the merge and fill properly https://github.com/Talend/ui/wiki/BREAKING-CHANGE

Reply:
👌